ui/panels-plugin/plugin.json

```diff
@@ -36,6 +36,14 @@
           "name": "Empty Chart",
           "description": ""
         }
+      },
+      {
+        "pluginType": "Panel",
+        "kind": "Markdown",
+        "display": {
```

Comment:
Are we also going to need ‘icon’ eventually (to show above the display name during add/edit flow)?

Reply:
Yeah, I think that's probably likely. Not exactly sure how that's going to work quite yet though since images are another thing (like we had with external CSS style sheets for react grid layout) that are handled by webpack loaders.

Reply:
Wonder if we could get by with inline SVG if the icons are simple enough, but yeah something to figure out later--I see they're gone in the most recent mockups anyways and it's just a dropdown